lblod/frontend-lokaal-mandatenbeheer
# Lmb 858 | Create an alert group for showing the multiple alerts in the IV page

## Description

In base PR i added a component to show a warning if the mandatees in an orgaan are above max or below min. This PR is extended from that to bundle those warnings into one alert that can display all messages.

## How to test

1. Go to IV page
2. See when the message needs to be triggered
3. Multiple messages should be show in a single alert component

## Links to other PR's

- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/443
